arbeitszeit_web/email/email_sender.py

```diff
@@ -44,4 +46,6 @@ def send_email(self, message: interface.Message) -> None:
         elif isinstance(message, interface.CooperationRequestEmail):
```

Comment:
As we discussed, I am slightly uneasy about this design pattern, but I like your proposal of (eventually) replacing this logic with a dictionary lookup.